core/src/main/java/com/novoda/noplayer/internal/listeners/PlayerListenersHolder.java

```diff
@@ -125,39 +119,39 @@ public void removeVideoSizeChangedListener(Player.VideoSizeChangedListener video
         videoSizeChangedListeners.remove(videoSizeChangedListener);
     }
 
-    public ErrorListeners getErrorListeners() {
+    public Player.ErrorListener getErrorListeners() {
```

Comment:
Use the interfaces, so that we can make the `ErrorListeners` etc package protected.